lib/topological_inventory/sync/inventory_upload_worker.rb

```diff
@@ -18,7 +20,55 @@ def perform(message)
         payload = JSON.parse(message.payload)
         return unless payload["service"] == "topological-inventory"
 
-        logger.info("#{payload}")
+        request = Typhoeus::Request.new(payload["url"])
+        response = request.run
+
+        if response.return_code == :ok # response.success? doesn't work for file:///...
+          data = JSON.parse(response.body)
```

Comment:
Do you need to handle the fact that this will be a tar.gz file?

Reply:
well, this is working version of PR, now processing uncompressed file